client.go

```diff
@@ -502,6 +502,33 @@ func (c *Client) LMTPData(statusCb func(rcpt string, status *SMTPError)) (io.Wri
 	return &dataCloser{c, c.Text.DotWriter(), statusCb}, nil
 }
 
+// SendMail is the low level email delivery agent.
```

Comment:
It would be nice to describe exactly what this function does, and what it doesn't do. Can take inspiration from the existing `SendMail` function. e.g. something like:

```go
// SendMail sends an email from address from, to addresses to, with message r.
//
// This function does not start TLS, nor does it perform authentication. Use
// StartTLS and Auth before-hand if desirable.
//
// The addresses in the to parameter are the SMTP RCPT addresses.
//
// The r parameter should be an RFC 822-style email with headers first, a blank
// line, and then the message body. The lines of r should be CRLF terminated.
```